test/lint/README.md

```diff
@@ -23,6 +23,12 @@ maintained:
 * for `src/crypto/ctaes`: https://github.com/bitcoin-core/ctaes.git (branch master)
 * for `src/crc32c`: https://github.com/google/crc32c.git (branch master)
 
+To do so, add the upstream repository as remote:
+
+```
+git remote add -f secp256k1 https://github.com/bitcoin-core/secp256k1.git
```

Comment:
I wish `-f` was called `--fetch` instead

Reply:
Changed.

Reply:
Oh, I didn't know `--fetch` existed. `git help remote` doesn't mention it, but it seems to work, so :man_shrugging:

Reply:
It does does show up when you do `git remote add`